### DISCUSSION TITLE
Some parts of GC Little Helper doesn't work

### DISCUSSION THREAD

**Discussion by Jisperveld** [2Abendsegler/GClh#1905] [Questions] [answered]
Since the update to version 0.11.5.2 some parts don't work properly anymore.
Especially the **Logging part**. Smileys aren't visible anymore. The placeholders do'nt show the underlying text. E.g.: in my logsignature I see #owner# and ##Found# in stead of the name of the cache-owner and the number of caches I found.
In a Message or Mail the placeholders do work correct.
Also in the **Personal Cache Note** section  "Adapt to the height of the personal cache note edit field?" isn't working.
I did verify my settings in the Configurator.
I'm using Opera, but I also tried Firefox en Chrome.

**Comment by capoaira** [comment]
The issues with the logging page are a known problem. GS has changed some internal things. The Bug is fixed with #1900 and will be released with the next version. Until then, you can use the old logging page as a workaround. Or you install the fix manually. The link for that is in the related Issue #1855 (https://github.com/2Abendsegler/GClh/issues/1855#issuecomment-914226916)

I cannot reproduce the Issue with the personal cache note. It works for me.
Can you have a look in your Console? (Right mouse click => Examine => Console) 

Edit: not "Examine", I mean "Inspect" (Translation Error, sorry)

**Reply by Jisperveld** [reply]
> "Can you have a look in your Console? (Right mouse click => Examine => Console)"

I don't know what you mean by this?

Number of finds works now, Owner not yet. I'll wait untill the new release.
Thanks for your help.

**Reply by capoaira** [reply]
> > "Can you have a look in your Console? (Right mouse click => Examine => Console)"
> 
> I don't know what you mean by this?

Sorry, my translation was wrong (I used Google translator).
In the Console, developers can get some information. You can open it if you do a right mouse click and choose `Inspect (Q)`. Then you can choose `Console`

**Comment by Jisperveld** [comment]
Sorry for my late reaction. I looked at the console and this is what it shows:
content: normal start event processing for df263d36-1ace-4d11-b0f8-b818fafdfd89 (1 to run)
content.js:98 content: Start ENV normally df263d36-1ace-4d11-b0f8-b818fafdfd89
VM14:70 Tampermonkey started
content.js:95 content: detected DOMContentLoaded df263d36-1ace-4d11-b0f8-b818fafdfd89
content.js:95 content: detected load df263d36-1ace-4d11-b0f8-b818fafdfd89
hotjar?v=M3hnFubH6Qo5k-eDoZ0qVoeedVIpl-ZejYn-UqgrWhc1:1 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
(anonymous) @ hotjar?v=M3hnFubH6Qo5k-eDoZ0qVoeedVIpl-ZejYn-UqgrWhc1:1
VM24:5 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
(anonymous) @ VM24:5
ga-audiences:1 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
ga-audiences:1 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
search?v=hSzVhttgk5Wv1ue5joB6EWJVdb9CK4SEdfFelNQPvjE1:1 --- jQuery Typeahead Debug ---
DevTools failed to load source map: Could not load content for https://www.geocaching.com/account/scripts/custom/message-center-header-widget.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by 2Abendsegler** [comment]
Hi,

I am looking at the feature for the personal cache note. It works for me, even with Opera.

Can you please deactivate all other scripts and add ons so that we can determine whether there is a conflict between them and the GClh? After this please try again.

Frank

**Reply by Jisperveld** [reply]
Everything is working now accept the placeholder #Owner# in my Cachelog signature.

**Reply by Jisperveld** [reply]
It is a while ago, but I still have a problem with the placeholder #Owner#. When I substitute the value like for instance #DateTime# it works perfect as you can see here : **06.01.2022 22:42 bedankt voor het leggen en onderhouden van deze cache**. Only #Owner# gives the following: **#Owner# bedankt voor het leggen en onderhouden van deze cache.**
My version of GCHelper is of 23.12.2021
I did try it with all other scripts  and add-ons deactivated but that dind't solve my problem.
I hope you can solve this.

**Comment by 2Abendsegler** [comment]
Hi,

I have checked it <del>one</del> _on_ both log forms with and without a draft. It works for me on firefox.

Can you please try a log with another cache?
Can you please look to the console? You can open it if you do a right mouse click and choose `Inspect (Q)`. Then you can choose `Console`. Please post the content again.
Perhaps you can post also a screenshot with the problem? Perhaps we can see something special. I have no other idea in the moment.

Cheers
Frank

**Comment by Jisperveld** [comment]
I use Opera instead of Firefox. Everything worked fine until September 10 - 2021. 
Here are my configuration setting of GCHelper for the log part:
![Placeholders for Log](https://user-images.githubusercontent.com/90464262/148606248-2dbea0a2-91af-4b74-ab9c-f3214c3627c4.png)

And this is the result regardless wether I log inline or not:
![Log with placeholders](https://user-images.githubusercontent.com/90464262/148606750-cd1a3505-6b7c-40d2-b8b2-e58c8fd37f5c.png)

And this is the listing of the Console. I disabled the Ad-blocker of Opera so I don't know why there is something about an Adblocker in the console:
content: normal start event processing for b26ad4da-cab8-4605-bab8-f8ea34725e3e (1 to run)
content.js:98 content: Start ENV normally b26ad4da-cab8-4605-bab8-f8ea34725e3e
VM14:70 Tampermonkey started
content.js:95 content: detected DOMContentLoaded b26ad4da-cab8-4605-bab8-f8ea34725e3e
content.js:95 content: detected load b26ad4da-cab8-4605-bab8-f8ea34725e3e
GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt:1 Mixed Content: The page at 'https://www.geocaching.com/geocache/GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt' was loaded over HTTPS, but requested an insecure element 'http://b.tile.openstreetmap.org/11/1055/680.png'. This request was automatically upgraded to HTTPS, For more information see https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html
GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt:1 Mixed Content: The page at 'https://www.geocaching.com/geocache/GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt' was loaded over HTTPS, but requested an insecure element 'http://c.tile.openstreetmap.org/11/1056/680.png'. This request was automatically upgraded to HTTPS, For more information see https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html
GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt:1 Mixed Content: The page at 'https://www.geocaching.com/geocache/GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt' was loaded over HTTPS, but requested an insecure element 'http://c.tile.openstreetmap.org/11/1055/681.png'. This request was automatically upgraded to HTTPS, For more information see https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html
GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt:1 Mixed Content: The page at 'https://www.geocaching.com/geocache/GC9EKBM_filmlocatie-da-vinci-code-in-stiphout#gclhLogIt' was loaded over HTTPS, but requested an insecure element 'http://a.tile.openstreetmap.org/11/1056/681.png'. This request was automatically upgraded to HTTPS, For more information see https://blog.chromium.org/2019/10/no-more-mixed-messages-about-https.html
ga_exp.js?utmxkey=3682814-19&utmx=&utmxx=&utmxtime=1641588774089&utmxhash=gclhLogIt:1 Google Analytics Content Experiments are no longer supported: (https://support.google.com/analytics/answer/9366791?hl=en). Please uninstall http://www.google-analytics.com/ga_exp.js from this page.
(anonymous) @ ga_exp.js?utmxkey=3682814-19&utmx=&utmxx=&utmxtime=1641588774089&utmxhash=gclhLogIt:1
www.google.com/ads/ga-audiences?t=sr&aip=1&_r=4&slf_rd=1&v=1&_v=j96&tid=UA-2020240-1&cid=1389808421.1611350020&jid=354863049&_u=SCCAAEIgAAAAAC~&z=1349555262:1 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
www.google.nl/ads/ga-audiences?t=sr&aip=1&_r=4&slf_rd=1&v=1&_v=j96&tid=UA-2020240-1&cid=1389808421.1611350020&jid=354863049&_u=SCCAAEIgAAAAAC~&z=1349555262:1 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
vendor?v=Ve63OM4s2fukdNdr-1HHH52xEtz-mLD0P1rztmVirwM1:3 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
insertScriptTag @ vendor?v=Ve63OM4s2fukdNdr-1HHH52xEtz-mLD0P1rztmVirwM1:3
content.js:117 content: normal start event processing for 5fe3542b-d1b4-42b1-8836-9036201c3d44 (1 to run)
content.js:98 content: Start ENV normally 5fe3542b-d1b4-42b1-8836-9036201c3d44
VM86:70 Tampermonkey started
content.js:95 content: detected DOMContentLoaded 5fe3542b-d1b4-42b1-8836-9036201c3d44
content.js:95 content: detected load 5fe3542b-d1b4-42b1-8836-9036201c3d44
vendor?v=sm_YngPqc5sXtcYeb4JGAjlNWjB9dM5GQCdS1cGsgco1:2 Failed to load resource: net::ERR_BLOCKED_BY_ADBLOCKER
insertScriptTag @ vendor?v=sm_YngPqc5sXtcYeb4JGAjlNWjB9dM5GQCdS1cGsgco1:2

I hope this will give you something towards a solution.

**Comment by 2Abendsegler** [comment]
Unfortunately I cannot see any problem. I installed Opera, Tampermonkey and the GClh and used them in the standard configurations. I customized the "Cache log signature" with #Owner# and I tried to do a log for the specified cache. It works fine.

Please send me your configuration of the GClh:
Please change your home coordinates first. Then please go to the GClh II Sync, there is a link to it in your [geocaching dashboard](https://www.geocaching.com/account/dashboard). Select "Manual" and then "Download". Please send me the file to my email address <del>deleted</del>

I will then install your configuration file on myself and hope that a problem then arises.

**Comment by Jisperveld** [comment]
I just sent you an email.

**Comment by 2Abendsegler** [comment, marked as answer]
If you know, then it's easy. But I didn't know either and had to analyze the coding. 🙄
I opened an issue (#2005) to improve the description of the parameter. It is not well described and leads to misunderstandings.

The solution for you: 
Please activate the parameter [Take also owner pseudonym to replace placeholder owner](https://www.geocaching.com/my/#GClhShowConfig#a#settings_show_pseudo_as_owner).

Nice weekend,
Frank

**Reply by Jisperveld** [reply]
Great!! Indeed if one knows, it's simple. 
I checked it and it works!!
Many thanks
